src/test/java/org/opensearch/security/test/SingleClusterTest.java

```diff
@@ -135,6 +138,31 @@ protected void setupGenericNodes(List<Settings> nodeOverride, List<Boolean> sslO
                 clusterConfiguration);
     }
 
+    //Wait for the security plugin to load roles.
+    public void waitForInit(Client client) throws Exception {
+        int maxRetries = 3;
+        Optional<Exception> retainedException = Optional.empty();
+        for (int i = 0; i < maxRetries; i++) {
+            try {
+                client.admin().cluster().health(new ClusterHealthRequest()).actionGet();
```

Comment:
Isn't a value returned that should be checked, could this value be something like not healthy/not up?